src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__finance_accounting_people_model.sql

```diff
@@ -14,10 +14,20 @@ with
             and extract(year from date_day) >= 2016
     ),
 
+    additional_earnings_clean as (
+        select distinct
```

Comment:
avoid using distinct. what's getting deduplicated here? can it be done on the staging table?

Reply:
leave as-is for now, @kevinverhoff needs to follow up w/ HR if there's a data quality issue